Source/WSTagView.swift

```diff
@@ -25,6 +29,46 @@ open class WSTagView: UIView, UITextInputTraits {
             setNeedsDisplay()
         }
     }
+    
+    open var imagePlaceholder = UIImage(systemName: "person.circle.fill") {
+        didSet {
+            if imageURL == nil {
+                imageView.image = imagePlaceholder
+                updateFrame()
+                setNeedsDisplay()
+            }
+        }
+    }
+    
+    open var imageURL: URL? {
+        didSet {
+            imageView.kf.setImage(with: imageURL, placeholder: imagePlaceholder)
+            updateFrame()
+            setNeedsDisplay()
+        }
+    }
+    
+    open var isImageHidden: Bool = true {
+        didSet {
+            imageView.isHidden = isImageHidden
+            setNeedsDisplay()
+            setNeedsLayout()
+        }
+    }
+    
+    open var imageSize: CGSize = CGSize(width: 22.0, height: 22.0) {
```

Comment:
Why a default of (22,22)?

Reply:
Also, is imageSize ever set or is it an internal variable?

Reply:
I picked the default size because that's what we're using in `memo-ios`, do you think I should calculate the `imageSize` based on `UILabel`?

It's internal now but I should add the ability to set it in `WSTagsField`